content/applications/general/email_communication/email_template.rst

```diff
@@ -64,35 +141,35 @@ website of the company.
    the same (the web.base.url key value), whatever the company it's shared from, this is a known
    limitation!
 
-On the other hand, sales orders made by a customer on one of your Odoo e-commerce websites have a
-link established with the website from which the order was made. As a result, the e-mail sent for
-the sales orders uses the domain name defined for the corresponding website to generate the links.
+On the other hand, sales orders made by a customer on an Odoo e-commerce website have a link
+established with the website from which the order was made. As a result, the e-mail sent for sales
+orders uses the domain name defined for the corresponding website to generate the links.
 
-For more information about how to configure your domains, we invite you to check :doc:`our domain name
-documentation </administration/maintain/domain_names>`.
+For more information about how to configure domains, check out :doc:`our domain name documentation
+</administration/maintain/domain_names>`.
 
 Updating translations within email templates
 --------------------------------------------
 
-Email templates are automatically translated. Changing the translations shouldn’t be necessary.
-However, if for a specific reason you’d like to change some of the translations, this can be done.
+Email templates are automatically translated. Changing the translations shouldn't be necessary.
+However, if for a specific reason, some of the translations need to be changed, it can be done.
 
-Like any modification in the code, keep in mind that modifications that aren’t done correctly (for
-example modifications leading to bad syntax) can break the template, as a result, the template
-will appear blank.
+Like any modification in the code, keep in mind that if they aren't done correctly (for example,
+modifications leading to bad syntax), it can break the template, and as a result, the template will
+appear blank.
```

Comment:
```suggestion
However, like any modification in the code, if they aren't done correctly (for example, modifications leading to bad syntax), it can break the template, and as a result, the template will appear blank.
```